src/main/resources/templates/charsheet.html

```diff
@@ -15,21 +15,23 @@
 			admin = [[${admin}]];
 			urlPrefix = [[${urlPrefix}]];
     </script>
-    <link rel="stylesheet" href="/css/charsheet.css"
-          th:href="@{/css/charsheet.css}" media="screen" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/charsheet.css"
+          th:href="@{charsheet.css}" media="screen" />
```

Comment:
Let's make absolutely sure we put this back... 🙂 
```suggestion
          th:href="@{/css/charsheet.css}" media="screen" />
```